Examples/Blink/CMakeLists.txt

```diff
@@ -37,19 +37,23 @@ set(STM32CUBEMX_GENERATED_FILES
         Core/Inc/stm32f3xx_it.h

```

Comment:
It won't let me comment on the right line but in order to get this to build I had to add the i2c hal driver files to the list above

Reply:
is it weird that I didn't notice this issue when I built?

Reply:
Yeah, only thing I could think of is that maybe you didn't run the CMake command again after making changes to the CMakeList